web-app/src/pages/tournament/TournamentCreator.vue

```diff
@@ -225,6 +253,10 @@ export default class TournamentCreator extends Vue {
   }
 
   private async submitAddTournament() {
+    const dateToSubmit = new Date(Date.parse(this.startDateString));
+    // console.log(dateToSubmit);
+    // console.log(this.days);
+    // console.log(this.frequency)
```

Comment:
.

Reply:
leaving comments for now, these are not connected to api yet